liqd/adhocracy3
# Service Konto backend

Testing without Frontend:

- in `etc/development.ini`, set `adhocracy.service_konto.enabled = true`.
- Create a Service Konto-Account: [https://skbref.verwalt-berlin.de/skb/FVP/Login/Anmeldung.aspx](https://skbref.verwalt-berlin.de/skb/FVP/Login/Anmeldung.aspx)
- Login to Service-Konto and select mein.berlin Sevice, then "weiter". You will be redirected to mein.berlin.de with a token as a parameter.
- Extract the Token from the network tab of the browser. 
- Login with curl: `curl http://localhost:6541/login_service_konto -H 'Content-Type: application/json;charset=UTF-8' --data-binary '{"token":"TOKEN"}`
- Note that the token is valid only for 20 seconds
